### DISCUSSION TITLE
Change from polar chart to bar chart

### DISCUSSION THREAD

**Discussion by scodari** [gnmyt/MySpeed#765] [Ideas]
Hi, thank you for this great project! I'm a data analyst and I'm used to choose the right visual for a specific type of data. I noticed that test duration in the statistic view uses a polar chart and I find it very unreadable. Technically, the right type of chart for this kind of data is an hystogram but I see you are using the "react-chartjs-2" library and that doesn't have hystograms, but it has bar charts. It's still not perfect but way better than the polar chart. Would you consider changing the type of graph?

**Comment by gnmyt** [comment]
Hi, you're definitely right. The polar chart doesn't really fit there. I'm going to convert this discussion into an issue and implement your changes in the next update. Thanks for your feedback!

**Comment by gnmyt** [comment]
I implemented the new chart type in #769 so I'm going to close this thread. Thanks again for your suggestion.
